generator/src/main/java/com/scottlogic/deg/generator/generation/GenerationConfig.java

```diff
@@ -134,6 +134,18 @@ public String toString() {
         }
     }
 
+    public enum OutputDestination {
+        STREAM(Constants.Destinations.STREAM),
+        FILE(Constants.Destinations.FILE);
+        private final String destination;
+
+        OutputDestination(String destination) { this.destination = destination; }
```

Comment:
This can be a simple enum, and the constants can be removed if the destination is derived from the presence of a filename